libs/components/validation/src/lib/modules/validators/validators.ts

```diff
@@ -10,37 +11,71 @@ export class SkyValidators {
    * Validates email addresses in reactive forms. Add this validator directly to the form control
    * model in the component class. If users enter values that are not valid email addresses, the
    * validator throws an error. Since this is a sync validator, it returns a set of validation
-   * errors or `undefined` immediately when users enter values.
+   * errors or `null` immediately when users enter values.
    * @param control
    */
   public static email(control: AbstractControl): ValidationErrors | null {
     const value = control.value;
 
     if (!value) {
-      return undefined;
+      return null;
     }
 
     return SkyValidation.isEmail(value)
-      ? undefined
+      ? null
       : { skyEmail: { invalid: value } };
   }
 
   /**
    * Validates URLs in reactive forms. Add this validator directly to the form control model in
    * the component class. If users enter values that are not valid URLs, the validator throws an
-   * error. Since this is a sync validator, it returns a set of validation errors or `undefined`
+   * error. Since this is a sync validator, it returns a set of validation errors or `null`
    * immediately when users enter values.
    * @param control
```

Comment:
This `@param` should probably be removed. TypeDoc does a good job of determining the method's parameters when it generates the documentation.

Reply:
resolved in: [remove param from jsdoc comments](https://github.com/blackbaud/skyux/pull/201/commits/445a36e575afc25770140edcc5217ce6a4f36cc6)